msteams-platform/toolkit/debug-your-Teams-app-test-tool.md

```diff
@@ -0,0 +1,288 @@
+---
+title: Debug your Teams App Test Tool
+author: surbhigupta 
+description: In this module, learn how to debug your Teams App Test Tool and key features of Teams App Test Tool.
+ms.author: surbhigupta 
+ms.localizationpriority: high
+ms.topic: overview
+ms.date: 11/03/2023
+---
+
+# Teams App Test Tool
+
+> [!NOTE]
+>
+> Ensure the prerelease version of Teams Toolkit is installed. For more information, see [install a prerelease version](install-Teams-Toolkit.md#install-a-pre-release-version).
+
+Teams App Test Tool (Test Tool) is a component integrated into the Teams Toolkit. The tool assists users to debug, test, and refine the app design of a Teams bot application. It provides a web-based chat environment that replicates the behavior, appearance, and user experience of Microsoft Teams. 
+
+   :::image type="content" source="../assets/images/teams-toolkit-v2/debug/test-tool.png" alt-text="Screenshot shows the Teams App Test Tool Adaptive Card.":::
+
+## Explore Teams App Test Tool
+
+### Advantages
+
+The Teams App Test Tool offers the following benefits:
+
+1. **Sandbox environment**: Sandbox environment emulates the behavior, look and user experience of Teams.
+
+1. **Tunneling**: Tunnel service is not required as it runs locally.
+
+1. **Reduce local debugging**: Microsoft 365 Developer tenant and the app uploading permissions aren't required to debug the bot application.
+
+1. **Rapid inner-loop iterations**: Optimizes the process of making changes to the app design and bot logic without having to redeploy the bot application to the cloud.
+
+1. **Mock data and activities**: The Test Tool makes it easy to accomplish a test for complex scenarios such as **Send a welcome message when a new member joins the channel** by using mock data and built-in and custom activity triggers.
+
+1. **Reliable and trustable**: The Test Tool is reliable as the Bot application [Adaptive Card](../task-modules-and-cards/cards/cards-reference.md#adaptive-card) utilizes the same rendering technology as in Teams.
+
+1. **Integration with existing Teams bot applications**: The Test Tool can be integrated with existing Teams bot applications that are built with [Bot Framework SDK](https://dev.botframework.com/).
+
+1. **Support for different scopes**: The Test Tool supports testing in personal, team, and group chat scopes.
+
+### Limitations
+
+The following are the limitations:
+
+1. The Test Tool doesn't process the app manifest, which means features that are only enabled through the manifest won't be accessible.
```

Comment:
@MuyangAmigo how about moving it from limitations to advantages section? It's by design so that users can focus on their code logic from the beginning.

Reply:
I don't think we could really rephrase it as an advantage TBH. It really is a limitation where developers can't simply use a certain feature as they are in Teams Client (E.g. command menu)